CONTRIBUTING.md

```
@@ -2,99 +2,139 @@
 
-Contributions to Casanovo follow a standard GitHub contribution workflow:
+### Development workflow
 
-1. Create your own fork of the Casanovo repository on GitHub.
+Casanovo follows a standard GitHub workflow with a few important specifics:
 
-2. Clone your forked Casanovo repository to work on locally.
+1. Fork the repository
+2. Clone your fork locally
+3. **Create your branch from the `dev` branch** (not `main`):
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)